src/components/avatar/avatar.visualroute.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { Avatar } from 'ui-kit';
+import Avatar from '@commercetools-frontend/ui-kit/dist/esm/Avatar';
```

Comment:
Consumers of ui-kit could do an import rewrite, like we do for lodash

```js
// Normal import
import Avatar from '@commercetools-frontend/ui-kit/Avatar'

// Transformed to
import Avatar from '@commercetools-frontend/ui-kit/dist/esm/Avatar'
```

Reply:
Would we still over the combined entry?

Reply:
Actually the rewriting won't work, we should simply use the "full import path"